### ISSUE TITLE
When I upload svg with raster image embedded in it whether base64 or withing foreign obejct, image does not supported by editor. 

### ISSUE THREAD

**Issue by khanamirabes** [SVG-Edit/svgedit#931]
When I upload svg with raster image embedded in it whether base64 or withing foreign obejct, image does not supported by editor. 
If I uploaded a svg with raster image, I can see only svg elements in editor not raster image. Please let me know what is the issue or how can I make editor to support raster image also. 

I have attached the svg with raster image(base64). When I open this svg into browser, I can see raster image there but when I upload same svg in editor, raster image can not be seen. 

Please help. Thanks in advance.
![airport](https://github.com/SVG-Edit/svgedit/assets/88640508/0d4efe2f-4cb8-41fc-acf7-41b8205ba83d)

**Comment by jfhenon** [comment]
Indeed, SVGEdit does not support the 'href' followed by the image content. You may see in the console that the href was "sanitized"  with the following message  "sanitizeSvg: attribute href in element image not in whitelist is removed" 
Unfortunately, changing this has several side effects (mainly because SVGEdit supports only xlink:href that is now deprecated in favor of href as of SVG2). 
This is not really a bug but this definitely needs to be improved.
